src/resources/swagger.yaml

```diff
@@ -65,6 +65,10 @@ paths:
               schema:
                 example: 'program_id\tsubmitter_donor_id\tgender\tsubmitter_specimen_id\tspecimen_type\ttumour_normal_designation\tsubmitter_sample_id\tsample_type'
                 type: string
+            application/zip:
```

Comment:
I think we need a seperate path for the 'all' version, since it has a different return type. This also calls clear attention to this endpoint instead of declaring the value in the description.

Reply:
:+1: